.prettierrc

```diff
@@ -1,7 +1,7 @@
 {
   "endOfLine": "lf",
   "semi": false,
-  "singleQuote": false,
+  "singleQuote": true,
```

Comment:
At the moment, I myself rely solely on **ESLint** to auto-format the code instead of using **prettier**.
But as we progress, if the need arises, there may be possibility of making prettier work in parallel with ESLint, perhaps using **eslint-config-prettier** & **eslint-plugin-prettier**